src/machine/machined-varlink.c

```diff
@@ -757,9 +759,10 @@ static int manager_varlink_init_userdb(Manager *m) {
         if (r < 0)
                 return log_error_errno(r, "Failed to register varlink methods: %m");
 
-        r = sd_varlink_server_listen_address(s, "/run/systemd/userdb/io.systemd.Machine", 0666 | SD_VARLINK_SERVER_MODE_MKDIR_0755);
+        const char *path = "/run/systemd/userdb/io.systemd.Machine";
```

Comment:
Might as well make these defines or static if you're pulling them out like this, no point in making them regular local variables